package org.clevercastle.authforge.core.exception;

public class UserNotFoundException extends CastleException {
    public UserNotFoundException() {
    }
}
